README.md

```diff
@@ -1,33 +1,143 @@
 
-## Bug
+- [ArmoniK.Core](https://github.com/aneoconsulting/ArmoniK.Core) - The core logic of ArmoniK
+- [ArmoniK.Api](https://github.com/aneoconsulting/ArmoniK.Api) - The gRPC APIs 
+- [ArmoniK.Extensions.Csharp](https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp) - The C# extensions
+- [ArmoniK.Extensions.Cpp](https://github.com/aneoconsulting/ArmoniK.Extensions.Cpp) - The C++ extensions
 
-You are welcome to raise issues on GitHub. Please, read our [community guidelines](https://aneoconsulting.github.io/ArmoniK.Community/) before doing so.
 
-You can also send us a direct email at [armonik-support@aneo.fr](mailto:armonik-support@aneo.fr).
+## Releases
 
-## Acknowledge
+The latest ArmoniK releases are available on [GitHub](https://github.com/aneoconsulting/ArmoniK/releases) and [Docker Hub](https://hub.docker.com/u/dockerhubaneo).
+
+
+
+## Benchmarking  
+
+Performance benchmarks for ArmoniK are available here:
+
+[Benchmark results](https://aneoconsulting.github.io/ArmoniK/benchmarking/test-plan/)
+
+## Resources  
+
+### Documentation
+
+Please, read [documentation](https://aneoconsulting.github.io/ArmoniK/) for more information about ArmoniK.
+
+👉 [ArmoniK Architecture](https://aneoconsulting.github.io/ArmoniK/armonik)
+👉 [ArmoniK Versions](https://aneoconsulting.github.io/ArmoniK/armonik#versions)
+👉 [ArmoniK Installation](https://aneoconsulting.github.io/ArmoniK/installation)
+👉 [ArmoniK Configuration](https://aneoconsulting.github.io/ArmoniK/guide/how-to/how-to-configure-authentication)
+👉 [ArmoniK Performance](https://aneoconsulting.github.io/ArmoniK/benchmarking/test-plan)
+
+👉 [Blog]   
+👉 [YouTube videos]
+
+## Contributing
+
+ArmoniK is open source and welcomes contributions. See the our [community guidelines](https://aneoconsulting.github.io/ArmoniK.Community/) before doing so.
+
+- Report bugs by [opening an issue](https://github.com/aneoconsulting/ArmoniK/issues) 
+- Suggest new features by [opening an issue](https://github.com/aneoconsulting/ArmoniK/issues)
+- Improve documentation by submitting a pull request
+- Find good first issues to work on with the [good first issue](https://github.com/aneoconsulting/ArmoniK/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) tag 
+- You can also send us a direct email at [armonik-support@aneo.fr](mailto:armonik-support@aneo.fr).
 
-This project was funded by AWS and started with their [HTCGrid project](https://awslabs.github.io/aws-htc-grid/).
 
 ## License
 
-[Apache License, Version 2.0](https://github.com/aneoconsulting/ArmoniK/blob/main/LICENSE)
+**ArmoniK.Core** is licensed under the [AGPL v3](https://github.com/aneoconsulting/ArmoniK.Core/blob/main/LICENSE) license.  
+
+Other components are under the [Apache 2](https://github.com/aneoconsulting/ArmoniK/blob/main/LICENSE) license.
+
+[Organization 1]: https://armonik.fr
+[Organization 2]: https://aneo.fr   
+[Blog]: Soon
+[YouTube videos]: https://example.com
```

Comment:
If it does not exist this should be erased

Reply:
I agree